### PR TITLE
Start scan series labels and more

### PR DESCRIPTION
**Pull request by local-minimum**
- Adds some nice category labels to jobs.
- Adds a progress bar to running jobs
- Remakes the info about the job in to a nice table
- Adds sorting of jobs by category and then running by reverse chronological completion date and completed by reverse chronological started date.
- Makes the page live update every 10s
screenshot from 2018-03-14 11-12-49

### CODE REVIEW

File: scanomatic/ui_server_data/js/specs/components/ScanningRoot.spec.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { shallow } from 'enzyme';
 import React from 'react';
 
 import './enzyme-setup';
-import ScanningRoot from '../../src/components/ScanningRoot';
+import ScanningRoot, { getStatus, jobSorter } from '../../src/components/ScanningRoot';
```
Comment (skymandr) [inline scanomatic/ui_server_data/js/specs/components/ScanningRoot.spec.jsx]: why the curlies?

File: scanomatic/ui_server_data/js/src/components/ScanningJobPanel.jsx [hunk 2]
```diff
@@ -4,6 +4,19 @@ import React from 'react';
 
 import SoMPropTypes from '../prop-types';
 
+export function duration2milliseconds(duration) {
+    if (duration) {
+        return (duration.minutes * 60000) + (duration.hours * 3600000) + (duration.days * 86400000);
```
Comment (skymandr) [inline scanomatic/ui_server_data/js/src/components/ScanningJobPanel.jsx]: isn't there a builtin for this?

File: scanomatic/ui_server_data/style/experiment.css [hunk 3]
```diff
@@ -15,6 +15,15 @@ div.panel button {
   margin: 0.25em;
 }
 
+div.panel-heading h3 {
```
[the remark this answers is not in the document]
Comment (skymandr) [reply scanomatic/ui_server_data/style/experiment.css]: hmm?

File: scanomatic/ui_server_data/js/specs/components/ScanningRoot.spec.jsx (hunk 1 above)
Comment (local-minimum) [reply scanomatic/ui_server_data/js/specs/components/ScanningRoot.spec.jsx]: That's the syntax, `ScanningRoot` is the default export the others are other exports.

File: scanomatic/ui_server_data/js/src/components/ScanningJobPanel.jsx (hunk 2 above)
Comment (local-minimum) [reply scanomatic/ui_server_data/js/src/components/ScanningJobPanel.jsx]: somewhat, updated it.

**Pull request merged**